src/main/java/RedisSessionManagerT8.java

```
@@ -0,0 +1,298 @@
+        return metadata;
+    }
+
+    /**
+     * Get attributes from a Session
+     * @param session
+     * @return
+     */
+    private Hashtable<String, Object> getAttributes(Session session){
+        StandardSession standardSession = (StandardSession)session;
+        Hashtable<String, Object> attributes = new Hashtable();
+        for (Enumeration<String> enumerator = standardSession.getAttributeNames(); enumerator.hasMoreElements();) {
+            String key = enumerator.nextElement();
+            attributes.put(key, standardSession.getAttribute(key));
+        }
+        return attributes;
+    }
+
+    //------------------------------------------------------------------------------------------------------------------
+    // END Utils
+    //------------------------------------------------------------------------------------------------------------------
+
+
+    //------------------------------------------------------------------------------------------------------------------
+    // START REDIS
+    //------------------------------------------------------------------------------------------------------------------
+
+
+    /**
+     * Method to saved session in redis
+     * @param session
+     * @return
+     * @throws IOException
+     */
+    protected boolean saveToRedis(Jedis jedis, Session session) throws IOException {
+        boolean errorSaving = true;
+        //Get the session Id
+        String id = session.getId();
+        Hashtable<String, Object> metadata = getMetadata(session);
+        Hashtable<String, Object> attributes = getAttributes(session);
+        try{
+            jedis.set((id + REDIS_METADATA_KEY).getBytes(), Base64.getEncoder().encode(SerializationUtils.serialize(metadata)));
+            jedis.set((id + REDIS_ATTRIBUTES_KEY).getBytes(), Base64.getEncoder().encode(SerializationUtils.serialize(attributes)));
+            errorSaving = false;
+        }catch (Exception e){
+        }
```

Don't ever just swallow errors. Tihs is evil.